### ISSUE TITLE
Custom SSL certificates support 

### ISSUE THREAD

**Issue by 0x366** [anhnmt/backup-pg2minio#16]
Hello, thank you for the nice util!

Could you add an option to pass a custom certificate to docker ? I have minio running on private vpn domain:

```
`/app/pg_dump.sql.gz` -> `minio/postgres/postgres/postgres_2024-01-18T15:25:28Z.sql.gz`
mc: <ERROR> Failed to copy `/app/pg_dump.sql.gz`. Get "https://<private-domain>/postgres/?location=": tls: failed to verify certificate: x509: certificate signed by unknown authority
2024-01-18T15:25:28Z ERR minio.go:31 > Failed to copy error="exit status 1"
2024-01-18T15:25:28Z PNC root.go:31 > Failed to start backup error="exit status 1"
```

**Comment by anhnmt** [comment]
I will add this option in the next version

**Comment by anhnmt** [comment]
@0x366 I have published v0.3.3, hope it helps you
